examples/ssl-verification/badssl.json

```diff
@@ -35,6 +35,28 @@
           }
         ]
       }
+    },
+    {
+      "id": "client-cert",
+      "backend_version": 1,
+      "proxy": {
+        "api_backend": "https://client.badssl.com",
+        "backend": {
+          "endpoint": "https://echo-api.3scale.net"
```

Comment:
?

Reply:
Did you use the echo to test if the cert is sent to the backend too when the reuse ssl session option is enabled?

Reply:
Yep. But I could not figure it out from the debug logs. Haven't done wireshark yet.